addon/utils/css-styleify.js

```diff
@@ -1,17 +1,11 @@
 import Ember from 'ember';
 
-const {
-  isPresent,
-  String: {
-    dasherize,
-    htmlSafe
-  }
-} = Ember;
+const { isPresent, String: { dasherize, htmlSafe } } = Ember;
```

Comment:
This also looks a bit weird. But ES6 module imports fix this as well.

Which reminds me - we should upgrade the code base to ES6 module imports. :grin: